### ISSUE TITLE
Allow more configuration for itch.io parser

### ISSUE THREAD

**Issue by rlach** [doZennn/steam-rom-manager#380]
I've installed my itch.io under proton on linux, so I could play and install windows games using it. 

I can set itch.io parser to the proper folder, and the parser will find the games. The parser reads games from the .db file inside the itch.

The problem is, the games are resolved to windows folder like:
`C:\users\steamuser\AppData\Roaming\itch\apps\towerfall/TowerFallAscension_Itch/TowerFall.exe`

And I need to replace them with the proper linux path in order for them to actually point into the proper folder.

It would also help a lot if I could set the proton version to be used, or if we could at the very least set the command line arguments.

Basically, what I need is Executable Configuration section from the other parsers, but available under itch.io.

I tried configuring it trough userConfigurations file, but the rom manager overwrites my manual changes there, so there's no way currently to achieve what I need. I also tried just using a glob parser, but because I can't parse the .db file this way I end up with some false hits for the .exe search, no matter what kind of regex I'll use.

**Comment by cbartondock** [comment]
Fix to be merged soon https://github.com/SteamGridDB/steam-rom-manager/pull/393

**Comment by cbartondock** [comment]
Merged into master.

Also don't touch the user configurations file. That isn't something you can just change fields in willy nilly, it is SRM's storage file for its own settings. If you break it, SRM will stop working.
